src/utils/works.ts

```diff
@@ -53,5 +54,137 @@ export const getWorkUrl = ({
     workUrl += `/chapters/${chapterId}`;
   }
 
-  return workUrl
+  return workUrl;
 };
+
+export const getWorkPage = async (workId: string) => {
+  return cheerio.load(
+    (
+      await axios.get<string>(`https://archiveofourown.org/works/${workId}`, {
+        // We set a cookie to bypass the Terms of Service agreement modal that appears when viewing works as a guest, which prevented some selectors from working. Appending ?view_adult=true to URLs doesn't work for chaptered works since that part gets cleared when those are automatically redirected.
+        headers: {
+          Cookie: "view_adult=true;",
+        },
+      })
+    ).data
+  ) as WorkPage;
+};
+
+export interface Work {
+  author: string;
+  title: string;
+  rating: string[];
+  warning: string[];
+  fandom: string[];
+  language: string;
+  published: string;
+  words: string;
+  chapters: string;
+  category: string[];
+  relationship: string[];
+  character: string[];
```

Comment:
I know that some of these objects are optional, but this interface makes them seem all required. See: https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties

Note that for arrays returning an empty one is preferable to making the property optional.